main.go

```diff
@@ -30,7 +30,8 @@ func main() {
 
 	// TODO(runcom): parametrize this when the bin starts
 	h := authz.NewHandler(novolume)
-	if err := h.ServeUnix("root", *flPluginSocket); err != nil {
+
+	if err = h.ServeUnix("root", *flPluginSocket); err != nil {
 		logrus.Fatal(err)
```

Comment:
it's inside the SDK so everyone can take advantage of this - see the email I just sent you